mods/cnc/maps/gdi05c/rules.yaml

```diff
@@ -0,0 +1,169 @@
+World:
+	LuaScript:
+		Scripts: campaign-global.lua, gdi05c.lua
+	MissionData:
+		Briefing: A GDI field base is under attack. They have fended off one attack but will not survive another.\n\nMove to the base, repair the structures and then launch a strike force to destroy the Nod base in the area.\n\nDestroy all Nod units and structures.
+		BackgroundVideo: podium.vqa
+		BriefingVideo: gdi5.vqa
+		StartVideo: seige.vqa
+		WinVideo: nodlose.vqa
+		LossVideo: gdilose.vqa
+	MapOptions:
+		ShortGameCheckboxLocked: True
+		ShortGameCheckboxEnabled: False
+	SmudgeLayer@SCORCH:
+		InitialSmudges:
+			46,24: sc5,0
+			45,21: sc3,0
+			26,11: sc3,0
+			25,10: sc4,0
+			25,9: sc3,0
+			24,9: sc3,0
+			23,8: sc3,0
+			24,7: sc2,0
+			32,6: sc6,0
+			31,6: sc6,0
+			29,6: sc1,0
+			35,5: sc1,0
+			20,5: sc5,0
+			30,2: sc2,0
+			33,1: sc1,0
+	SmudgeLayer@CRATER:
+		InitialSmudges:
+			26,9: cr1,0
+			22,8: cr1,0
+			29,7: cr1,0
+			23,7: cr1,0
+			35,2: cr1,0
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: Difficulty
+		Values:
+			easy: Easy
+			normal: Normal
+			hard: Hard
+		Default: easy
```

Comment:
Default should be normal if we're wanting this to be standardized across missions.